### PR TITLE
Add Verified Data pages + obscuro Info endpoint

### PR DESCRIPTION
**Pull request by otherview**
### Why this change is needed

https://github.com/obscuronet/obscuro-internal/issues/2065

- Adds Verified Data pages with Sequencer Data + Know Contracts
- Adds the Info endpoint on obscuro

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

**Review by BedrockSquirrel — COMMENTED** [review]
LGTM generally but don't think we should be going to the enclave for static, public info - unless there's some future-proofing thing going on maybe?

**Review by zkokelj — APPROVED** [review]
LGTM!

File: go/enclave/enclave.go [hunk 1]
```diff
@@ -1388,6 +1388,13 @@ func (e *enclaveImpl) GetPublicTransactionData(pagination *common.QueryPaginatio
 	}, nil
 }
 
+func (e *enclaveImpl) Config() (*common.ObscuroEnclaveConfig, common.SystemError) {
```
[the remark this answers is not in the document]
Comment (otherview) [reply go/enclave/enclave.go:1391]: It does, because the enclaves has a different starting point than the host.

**Review by BedrockSquirrel — APPROVED** [review]
LGTM, hopefully we can get rid of that enclave call when we improve config

**Pull request merged**